build-system/test-configs/config.js

```diff
@@ -18,7 +18,7 @@
 const initTestsPath = ['test/_init_tests.js'];
 
 const fixturesExamplesPaths = [
-  'test/fixtures/*.html',
+  'test-bin/test/fixtures/*.html',
   {
     pattern: 'test/fixtures/served/*.html',
```

Comment:
Does this need to be updated too?

Reply:
it does not, so those files in served are actually re-iframed inside the outer fixtures. if i change these i would need to change the path to a few test files. these files also do not need to be transformed and can be thought of as non cdn files.